### PR TITLE
Alek/chatupdate

### PR DESCRIPTION
**Pull request by Alek99**

### CODE REVIEW

File: webui/webui/components/chat.py [hunk 1]
```diff
@@ -56,52 +58,55 @@ def chat() -> rx.Component:
...
-                            _hover={"border_color": styles.accent_color},
-                            style=styles.input_style,
+                    rx.hstack(
+                        rx.radix.text_field.root(
+                            rx.radix.text_field.input(
+                                placeholder="Type something...",
+                                id="question",
+                                width="30em",
+                            ),
+                            rx.radix.text_field.slot(
```
Comment (picklelo) [inline webui/webui/components/chat.py:72]: Looks nice visually, but I wonder if we should simplify this to just an `rx.input` as the interface is already intuitive

File: webui/webui/components/chat.py [hunk 2]
```diff
@@ -14,23 +16,23 @@ def message(qa: QA) -> rx.Component:
     Returns:
         A component displaying the question/answer pair.
     """
-    return rx.chakra.box(
-        rx.chakra.box(
+    return rx.box(
+        rx.box(
```
Comment (picklelo) [inline webui/webui/components/chat.py:20]: Should we change the inner components to `rx.text`

File: webui/webui/components/chat.py [hunk 3]
```diff
@@ -14,23 +16,23 @@ def message(qa: QA) -> rx.Component:
...
             margin_top="1em",
         ),
-        rx.chakra.box(
+        rx.box(
             rx.chakra.text(
                 qa.answer,
-                bg=styles.accent_color,
-                shadow=styles.shadow_light,
-                **styles.message_style,
+                background_color=rx.color("accent", 4),
```
Comment (picklelo) [inline webui/webui/components/chat.py:33]: Can we just use `color_scheme` here

File: webui/webui/components/chat.py [hunk 4]
```diff
@@ -56,52 +58,55 @@ def chat() -> rx.Component:
...
                 on_submit=State.process_question,
                 reset_on_submit=True,
-                width="100%",
             ),
             rx.chakra.text(
                 "ReflexGPT may return factually incorrect or misleading responses. Use discretion.",
-                font_size="xs",
-                color="#fff6",
                 text_align="center",
+                font_size=".75em",
```
Comment (picklelo) [inline webui/webui/components/chat.py:97]: Should we switch to Radix text and use the `size` prop

File: webui/webui/components/navbar.py [hunk 5]
```diff
@@ -1,68 +1,124 @@
...
+    )
+
+
+def modal(trigger) -> rx.Component:
+    """A modal to create a new chat."""
+    return rx.dialog.root(
+        rx.dialog.trigger(trigger),
+        rx.dialog.content(
+            rx.hstack(
+                rx.input(
```
Comment (picklelo) [inline webui/webui/components/navbar.py:61]: This should go in an `rx.form` once the overlay issues are fixed

File: webui/webui/components/navbar.py [hunk 6]
```diff
@@ -1,68 +1,124 @@
...
+        ),
+    )
+
+
+def navbar():
+    return rx.box(
+        rx.hstack(
+            rx.hstack(
+                rx.avatar(fallback="CC", variant="solid"),
+                rx.heading("Coca-Cola"),
```
Comment (picklelo) [inline webui/webui/components/navbar.py]: Do we want to merge this in as is? Maybe have Reflex by default and make it easy to customize the theme

**Review by picklelo — COMMENTED** [review]
looking nice!

File: webui/webui/components/navbar.py (hunk 6 above)
Comment (Alek99) [reply webui/webui/components/navbar.py]: Fixed

File: webui/webui/components/navbar.py (hunk 5 above)
Comment (Alek99) [reply webui/webui/components/navbar.py:61]: I was trying to reduce complexity in spots where I can, form is not really needed here as closing the dialog clears the output

File: webui/webui/components/chat.py (hunk 3 above)
Comment (Alek99) [reply webui/webui/components/chat.py:33]: Text color_schemes don't have backgrounds, so manually had to customize it here

File: webui/webui/components/chat.py (hunk 2 above)
Comment (Alek99) [reply webui/webui/components/chat.py:20]: fixed

File: webui/webui/components/chat.py (hunk 4 above)
Comment (Alek99) [reply webui/webui/components/chat.py:97]: This needs to be smaller than the default (smallest) 1 size, that why I had to resort to manually setting font size here

File: webui/webui/components/chat.py (hunk 1 above)
Comment (Alek99) [reply webui/webui/components/chat.py:72]: I like the look of this, I think it makes it look more professional and people can customize what message they want in tooltip for further instructions

File: webui/webui/components/navbar.py (hunk 5 above)
Comment (picklelo) [reply webui/webui/components/navbar.py:61]: Mostly so you can hit enter to submit instead of clicking the button

**Review by picklelo — APPROVED** [review]
looks good, update PR/commit title before merging

**Pull request merged**